### PR TITLE
feat: HW req for onprem installation.

### PR DESCRIPTION
**Pull request by M4itee**
##### Summary

Adding information about required hardware for the onprem installations.
##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: docs/netdata-cloud/netdata-cloud-on-prem/installation.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ The following components are required to install Netdata Cloud On-Prem:
 - **Helm** version 3.12+ with OCI Configuration (explained in the installation section)
 - **Kubectl**
 
+Minimum requirements for Netdata-Cloud are below:
```
Comment (Ancairon) [inline docs/netdata-cloud/netdata-cloud-on-prem/installation.md:13]:
```suggestion
The minimum requirements for Netdata-Cloud are:
```

File: docs/netdata-cloud/netdata-cloud-on-prem/installation.md [hunk 2]
```diff
@@ -10,6 +10,18 @@ The following components are required to install Netdata Cloud On-Prem:
 - **Helm** version 3.12+ with OCI Configuration (explained in the installation section)
 - **Kubectl**
 
+The minimum requirements for Netdata-Cloud are:
+- 4 CPU cores
+- 15GiB of memory
+- Cloud services are ephemeral
+
+Requirements for non-production Dependencies helm chart:
```
Comment (Ancairon) [inline docs/netdata-cloud/netdata-cloud-on-prem/installation.md:18]:
```suggestion
The requirements for the non-production Dependencies helm chart:
```

File: docs/netdata-cloud/netdata-cloud-on-prem/installation.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ The following components are required to install Netdata Cloud On-Prem:
...
+- 4 CPU cores
+- 15GiB of memory
+- Cloud services are ephemeral
+
+The requirements for the non-production Dependencies helm chart:
+- 8 CPU cores
+- 14GiB of memory
+- 160GiB for PVCs (SSD)
+
+> **_NOTE:_** Values for each component may vary depending on the type of load. The most intensive task, compute-wise, that the cloud needs to perform is the initial sync of directly connected agents. Requirements testing was done with 1,000 nodes directly connected to the cloud. If you plan on spawning hundreds of new nodes in a few minutes time window, Postgres is going to be the first bottleneck. For example, a 2 vCPU / 8 GiB memory / 1k IOPS database can handle 1,000 nodes without any problems if your environment is fairly steady, adding nodes in 10-30 batches (directly connected).
```
Comment (Ancairon) [inline docs/netdata-cloud/netdata-cloud-on-prem/installation.md:23]:
```suggestion
> **_NOTE:_** Values for each component may vary depending on the type of load. The most intensive task, compute-wise, that the Cloud needs to perform is the initial sync of directly connected Agents. The testing for these requirements was done with 1,000 nodes directly connected to the Cloud. If you plan on spawning hundreds of new nodes in a few minutes time window, Postgres is going to be the first bottleneck. For example, a 2 vCPU / 8 GiB memory / 1k IOPS database can handle 1,000 nodes without any problems if your environment is fairly steady, adding nodes in 10-30 batches (directly connected).
```

**Review by Ancairon — APPROVED** [review]
Looking good, did some grammar and uppercase edits :+1:

**Comment by M4itee** [comment]
actually I will change all of the "cloud" to "onprem" because with our current situation it is actually misleading

**Review by Ancairon — APPROVED** [review]

**Pull request merged**